notebooks/225-distilbert-sequence-classification/225-distilbert-sequence-classification.ipynb

```diff
@@ -0,0 +1,292 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from transformers import AutoTokenizer\n",
+    "import openvino.runtime as ov\n",
+    "import warnings\n",
+    "from pathlib import Path\n",
+    "import numpy as np\n",
+    "import time"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4dde5b43",
+   "metadata": {},
+   "source": [
+    "# Transformers Serialization"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1742fd4b",
+   "metadata": {},
+   "source": [
+    "Check out this link [https://huggingface.co/docs/transformers/serialization] to learn how to convert transformers to onnx format"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5db803ea",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python -m transformers.onnx -h"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4794f066",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "checkpoint = \"distilbert-base-uncased-finetuned-sst-2-english\"\n",
+    "serialize_command = f\"python -m transformers.onnx \\\n",
+    "    -m {checkpoint} \\\n",
+    "    --feature sequence-classification model/\"\n",
+    "! $serialize_command"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ae70bbf5",
+   "metadata": {},
+   "source": [
+    "## Initializing The Tokenizer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "782bbebf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tokenizer = AutoTokenizer.from_pretrained(\n",
+    "    pretrained_model_name_or_path=checkpoint\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8fff79fc",
+   "metadata": {},
+   "source": [
+    "# Model Optimizer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a0f48c1b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "onnx_model_path = \"model.onnx\"\n",
+    "MODEL_DIR = \"model/\"\n",
+    "MODEL_DIR = f\"{MODEL_DIR}\"\n",
+    "onnx_model_path = Path(MODEL_DIR) / onnx_model_path\n",
+    "\n",
+    "optimizer_command = f\"mo \\\n",
+    "    --input_model {onnx_model_path} \\\n",
+    "    --output_dir {MODEL_DIR} \\\n",
+    "    --model_name {checkpoint} \\\n",
+    "    --input input_ids,attention_mask \\\n",
+    "    --input_shape [1,128],[1,128]\"\n",
+    "! $optimizer_command"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "27cc074e",
+   "metadata": {},
+   "source": [
+    "Creating the input and output features for the inference"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e31a2644",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "warnings.filterwarnings(\"ignore\")\n",
+    "core = ov.Core()\n",
+    "ir_model_xml = str((Path(MODEL_DIR) / checkpoint).with_suffix(\".xml\"))\n",
+    "compiled_model = core.compile_model(ir_model_xml, \"AUTO\")\n",
+    "infer_request = compiled_model.create_infer_request()"
```

Comment:
please add some comments to code regarding API usage, what step responsible for what.
It is important not only provide some working pipeline, but also teach people how to use openvino

Reply:
Done